src/Microsoft.Sbom.Api/Workflows/Helpers/ExternalDocumentReferenceGenerator.cs

```diff
@@ -80,7 +88,17 @@ public async Task<GenerationResult> GenerateAsync()
                 }
             }
 
-            return new GenerationResult(totalErrors, jsonDocumentCollection.SerializersToJson, jsonArrayStarted);
+            var generationResult = new GenerationResult(totalErrors, jsonDocumentCollection.SerializersToJson, jsonArrayStartedForConfig);
+            foreach (var manifestInfo in manifestInfosFromConfig)
+            {
+                var config = sbomConfigs.Get(manifestInfo);
```

Comment:
I believe that `manifestInfosFromConfig` is unfiltered. Should we use a `TryGet` to protect against unregistered configs?